### PR TITLE
added the two readme files for warmup challenge

### PR DESCRIPTION
**Pull request by wattmatters**

### CODE REVIEW

File: 1_warmup/alexsimmons1964/01/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+does this make sense?
```
Comment (kotp) [inline 1_warmup/alexsimmons1964/01/README.md:1]: This really does make sense!

**Comment by patmaddox** [comment]
This pull request appears modify only your own files - nice work! Now it just needs final review from a human... (this comment was auto-generated :)

**Comment by patmaddox** [comment]
way to stay with us @AlexSimmons1964 ! and a big :+1: to @kotp for helping you out :)

**Comment by wattmatters** [comment]
Yes I appreciate the help. :+1:

**Comment by kotp** [comment]
You are welcome.  I did check your github profile and it looked like you did well committing for a block of time while going through some tutorial.  I am sure you will soon fill the gaps in your understanding of git, but it is not the simplest tool to use.

There is a pretty good introduction to [Git For Ages 4 And Up](https://www.youtube.com/watch?v=1ffBJ4sVUb4) conceptually in a video from a few years ago.

**Pull request merged**